src/Octoshift/Services/ReclaimService.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Octoshift.Models;
+using OctoshiftCLI;
+using OctoshiftCLI.Models;
+
+namespace Octoshift.Services
```

Comment:
Can we get rid of the dedicated `Services` folder. Technically all of our clients are services, so to avoid confusion let's place this alongside others in the root folder.

Reply:
done